Update README.md

lasomethingsomething opened the pull request:
Hi! Just a few little fixes here and there. Neat project; my ex-colleague randomly reminded me of it today. :)

Commit:
Update README.md

Hi! Just a few little fixes and nerding-out edits. Neat project; my ex-colleague randomly reminded me of it today. :)

willingc commented on README.md:
```diff
@@ -122,28 +122,27 @@ sentences from one open source project may be used to train the sentiment model
 The Stanford CoreNLP includes a way to retrain the neural network to recognize
 sentiment of sentence structures. You have to feed it a training set (their
-training set is ~8,000 sentences) and a development set that helps you tune
+training set is ~8,000 sentences) and a development set that helps you tunne1
```
~~tunne1~~ tune should remain as initially written

lasomethingsomething replied:
Aha, I think a stray 1 ended in there by accident, I saw "tune1," and turned it to tunnel. :)

willingc commented on README.md:
```diff
@@ -122,28 +122,27 @@ sentences from one open source project may be used to train the sentiment model
 
-FOSS Heartbeat's training set can be found in empathy/train.txt and FOSS
-Heartbeat's development set is found in empathy/dev.txt.
+You can find FOSS Heartbeat's training set in [empathy/train.txt](https://github.com/sarahsharp/foss-heartbeat/blob/master/empathy-model/train.txt) and its development set in [empathy/dev.txt](https://github.com/sarahsharp/foss-heartbeat/blob/master/empathy-model/dev.txt).
```
[`empathy/train.txt`]

[`empathy/dev.txt`]

willingc commented on README.md:
```diff
@@ -198,8 +197,8 @@ name. Since FOSS Heartbeat's model is trained to recognize empathy and praise
 
-You can use substitutions.txt to change word sentiment labels in the sentences
-from the default sentiment model. It involves stripping the '%' off the vim
+You can use [substitutions.txt](https://github.com/sarahsharp/foss-heartbeat/blob/master/language/substitutions.txt) to change word sentiment labels in the sentences
```
[`substitutions.txt`]

willingc commented on README.md:
```diff
@@ -198,8 +197,8 @@ name. Since FOSS Heartbeat's model is trained to recognize empathy and praise
-from the default sentiment model. It involves stripping the '%' off the vim
+You can use [substitutions.txt](https://github.com/sarahsharp/foss-heartbeat/blob/master/language/substitutions.txt) to change word sentiment labels in the sentences
+from the default sentiment model. It involves stripping the '%' off the Vim
 substitution commands in substitution.txt, using the resulting file as a sed
```
If you have time, please put back-ticks around substitution.txt

lasomethingsomething replied:
Done. I also standardized around this a bit through the rest of the doc, and found another typo in a file reference.

willingc commented on README.md:
```diff
@@ -227,20 +226,20 @@ process sentences, we need to drop any inline code that is (most likely) to be
-We also convert any unicode emojis into their short-hand codes (as described on
-http://www.webpagefx.com/tools/emoji-cheat-sheet/), which make it easier on
+We also convert any Unicode emojis into their short-hand codes (as described at
+http://www.webpagefx.com/tools/emoji-cheat-sheet/), which makes it easier on
```
Should be "make" since the verb agrees with "emojis" which is plural.

lasomethingsomething replied:
As written it appears that "make" points back to "converting," so either "converting" makeS it easier or the *emojis make* it easier and the comma should disappear.

lasomethingsomething replied:
Fixed everything noted in the comments but this one; awaiting review. :)

willingc commented on README.md:
```diff
@@ -227,20 +226,20 @@ process sentences, we need to drop any inline code that is (most likely) to be
-It also takes time for the Stanford CoreNLP to load the models, so it is faster
-to write a bunch of text to a file, and use the -file command line option to
+It also takes time for the Stanford CoreNLP to load the models. It is faster
+to write a bunch of text to a file and use the -file command line option to
```
`- file` Put the -file within backticks

willingc commented:
Hi @LappleApple,

Thanks for the PR :smile: I love when folks improve documentation for the next user. :cake:

I've made a fix minor changes and comments. I'm happy to merge this once they are made. Thanks!

LappleApple commented:
Thanks, @willingc. I'm actually on a mission ... https://github.com/lappleapple/feedmereadmes

Commit:
Update README.md

Small fixes.

willingc commented on README.md:
```diff
@@ -129,11 +129,11 @@ emojis.
 turned into Penn Tree format.
 
-You can find FOSS Heartbeat's training set in [empathy/train.txt](https://github.com/sarahsharp/foss-heartbeat/blob/master/empathy-model/train.txt) and its development set in [empathy/dev.txt](https://github.com/sarahsharp/foss-heartbeat/blob/master/empathy-model/dev.txt).
+You can find FOSS Heartbeat's training set in [empathy/train.txt] and its development set in [empathy/dev.txt].
```
Sorry I wasn't clear enough on the change here and on Line 200. If you would be willing to update that would be great, otherwise, I can merge and edit in a separate PR.

I did want to have the markdown link remain but since we are referencing files it would be helpful to put the file in backticks within the square brackets like so:

```
You can find FOSS Heartbeat's training set in [`empathy/train.txt`](https://github.com/sarahsharp/foss-heartbeat/blob/master/empathy-model/train.txt) and its development set in [`empathy/dev.txt`](https://github.com/sarahsharp/foss-heartbeat/blob/master/empathy-model/dev.txt).
```

which should render like this:

You can find FOSS Heartbeat's training set in [`empathy/train.txt`](https://github.com/sarahsharp/foss-heartbeat/blob/master/empathy-model/train.txt) and its development set in [`empathy/dev.txt`](https://github.com/sarahsharp/foss-heartbeat/blob/master/empathy-model/dev.txt).

willingc reviewed (commented):
Thanks, one small clarification on my earlier comments.

Commit:
Update README.md

Commit:
Update README.md

Standardized sub-files linking and formatting, linking to sub-files on first mention.

willingc commented:
Thanks @LappleApple. Good luck with your project too! 🌺
